Editor/Resources/EditorWindow/Pages/LandingPage.uxml

```diff
@@ -0,0 +1,61 @@
+            <ui:Label name="LandingPageDescription" text="Amazon GameLift is a dedicated game server hosting solution that deploys, operates, and scales cloud servers for multiplayer games.&#10;Whether you’re looking for a fully managed solution, or just the feature you need, GameLift leverages the power of AWS to deliver the best latency possible, low player wait times, and maximum cost savings."/>
+        </ui:VisualElement>
+        <ui:VisualElement class="divider"/>
+        <ui:VisualElement class="separator separator--large separator--horizontal">
+            <ui:VisualElement class="card">
+                <ui:VisualElement class="card__container">
+                    <ui:Image name="LandingPageNoAccountImage" class="card__icon icon--large"/>
+                    <ui:Label name="LandingPageNoAccountCardText" class="card__text"
+                              text="I don&apos;t have an AWS account"/>
+                </ui:VisualElement>
+                <ui:Button class="button card__button button--large" text="Create an AWS Account   \uf08e"
+                           name="CreateAccount"/>
+            </ui:VisualElement>
+            <ui:VisualElement class="card">
+                <ui:VisualElement class="card__container">
+                    <ui:Image name="LandingPageAccountImage" class="card__icon icon--large"/>
+                    <ui:Label name="LandingPageAccountCardText" class="card__text" text="I have an AWS account"/>
+                </ui:VisualElement>
+                <ui:Button class="button button--primary button--large" text="Add a new Profile" name="AddProfile"/>
+            </ui:VisualElement>
+        </ui:VisualElement>
+        <ui:VisualElement>
+            <ui:Label name="LandingPageSampleHeader" class="page__subtitle" text="Try our Sample Game"/>
+            <ui:VisualElement class="page__content separator separator--large separator--horizontal">
+                <ui:Image name="LandingPageSampleImage" class="sample__image"/>
+                <ui:VisualElement class="separator separator--thin">
+                    <ui:Label name="LandingPageSampleDescription" text="Test out our sample multiplayer game with GameLift Anywhere."/>
+                    <ui:Button name="DownloadSampleGame" class="button sample__button button--small" text="Import"/>
+                </ui:VisualElement>
+            </ui:VisualElement>
+        </ui:VisualElement>
+        <ui:VisualElement>
+            <ui:Label name="LandingPageCaseStudiesHeader" class="page__subtitle" text="[Case Studies]"/>
+            <ui:VisualElement class="separator separator--large separator--horizontal">
+                <ui:VisualElement class="separator">
+                    <ui:Image name="CaseStudy1" class="case-study__image"/>
+                    <ui:Label name="LandingPageCaseStudyDescription" text="Lorem ipsum dolor sit amet, \n consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."/>
```

Comment:
These names need to be different between the three in order for us to assign translations and links

Reply:
Don't the links need to be different for us to attach the correct URLs?

Reply:
Not blocking the PR on this as it will come up in the PR to add the case study links